nni/retiarii/hub/pytorch/autoformer.py

```diff
@@ -80,61 +66,68 @@ class RelativePositionAttention(nn.Module):
     and keys in self-attention modules.
     """
     def __init__(
-            self,
-            embed_dim,
-            fixed_embed_dim,
-            num_heads,
-            attn_drop=0.,
-            proj_drop=0,
-            rpe=False,
-            qkv_bias=False,
-            qk_scale=None,
-            rpe_length=14) -> None:
+            self, embed_dim, num_heads,
+            attn_drop=0., proj_drop=0.,
+            qkv_bias=False, qk_scale=None,
+            rpe_length=14, rpe=False,
+            head_dim=64):
         super().__init__()
         self.num_heads = num_heads
-        head_dim = embed_dim // num_heads
+        # head_dim is fixed 64 in official autoformer. set head_dim = None to use flex head dim.
+        self.head_dim = head_dim or (embed_dim // num_heads)
         self.scale = qk_scale or head_dim ** -0.5
-        self.qkv = nn.Linear(embed_dim, embed_dim * 3, bias=qkv_bias)
+        
+        # When head nums changed, the activate params position is important. 
+        # For example, when head nums increse from 2 to 3.
+        # If nn.Linear(embed_dim, (head_dim * num_heads) * 3) used, 
```

Comment:
It's actually the same as what was implemented in MixedMultiheadAttention.

If you want to write a few more notes here, please add more contexts (at least "refer to MixedMultiheadAttention for details and blabla"), to avoid causing confusions to first-time readers.